### ISSUE TITLE
MX Anywhere was crashing Solaar when activated until `killall solaar`

### ISSUE THREAD

**Issue by dragon788** [pwr-Solaar/Solaar#468]
Ubuntu 18.04.1 with all software updates as of 10/23/18.
Receiver updated with `fwupdmgr update`.

```
14:54:12,239    ERROR [ReceiverListener:hidraw5] logitech_receiver.listener: processing Notification(3,02,00,0006700200000000000000)
Traceback (most recent call last):
  File "/usr/share/solaar/lib/logitech_receiver/listener.py", line 185, in run
    self._notifications_callback(n)
  File "/usr/share/solaar/lib/solaar/listener.py", line 207, in _notifications_handler
    _notifications.process(dev, n)
  File "/usr/share/solaar/lib/logitech_receiver/notifications.py", line 54, in process
    return _process_device_notification(device, status, notification)
  File "/usr/share/solaar/lib/logitech_receiver/notifications.py", line 105, in _process_device_notification
    return _process_hidpp10_custom_notification(device, status, n)
  File "/usr/share/solaar/lib/logitech_receiver/notifications.py", line 130, in _process_hidpp10_custom_notification
    if n.sub_id == _R.illumination:
AttributeError: 'NamedInts' object has no attribute 'illumination'
14:54:12,247    ERROR [ReceiverListener:hidraw5] logitech_receiver.listener: processing Notification(3,02,00,000BF00000000000000000)
Traceback (most recent call last):
  File "/usr/share/solaar/lib/logitech_receiver/listener.py", line 185, in run
    self._notifications_callback(n)
  File "/usr/share/solaar/lib/solaar/listener.py", line 207, in _notifications_handler
    _notifications.process(dev, n)
  File "/usr/share/solaar/lib/logitech_receiver/notifications.py", line 54, in process
    return _process_device_notification(device, status, notification)
  File "/usr/share/solaar/lib/logitech_receiver/notifications.py", line 105, in _process_device_notification
    return _process_hidpp10_custom_notification(device, status, n)
  File "/usr/share/solaar/lib/logitech_receiver/notifications.py", line 130, in _process_hidpp10_custom_notification
    if n.sub_id == _R.illumination:
AttributeError: 'NamedInts' object has no attribute 'illumination'
14:54:12,575     INFO [ReceiverListener:hidraw5] solaar.listener: status_changed <PairedDevice(3,1017,Anywhere MX)>: paired offline, Battery: 70% (0) 
14:54:12,578  WARNING [ReceiverListener:hidraw5] logitech_receiver.notifications: <PairedDevice(3,1017,Anywhere MX)>: unrecognized Notification(3,42,01,0000000000000000000000)
```

**Comment by pfps** [comment]
Is this fixed by the patch fixing #531?

**Comment by pfps** [comment]
Closing due to no response from original poster.
